webpack/admindev.js

```diff
@@ -21,7 +21,7 @@ module.exports = merge.strategy( {
 	externals,
 	output: {
 		path: resolve( `${ __dirname }/../`, pkg.square1.paths.core_admin_js_dist ),
-		publicPath: `/wp-content/themes/core/${ pkg.square1.paths.core_admin_js_dist }`,
+		publicPath: `/${ pkg.square1.paths.core_admin_js_dist }`,
```

Comment:
Godd catch. I gapped when updating webpack here. I think you can just remove publicPath now altogether https://github.com/webpack/webpack/issues/11127 I did on my other projects and it works great.

Reply:
Ooo, Nice! I'll give that a shot here. Thank you!